moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h

```diff
@@ -118,7 +118,7 @@ moveit_msgs::Constraints constructGoalConstraints(const robot_state::RobotState&
  *
  * @return A full constraint message containing both constraints
  */
-moveit_msgs::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
+moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
```

Comment:
PoseStamped use needs to be adapted according to https://index.ros.org/doc/ros2/Contributing/Migration-Guide/
```suggestion
moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::msg::PoseStamped& pose,
```

Reply:
outside scope of this PR